src/Serilog/Formatting/Display/MessageTemplateTextFormatter.cs

```diff
@@ -82,11 +82,12 @@ public void Format(LogEvent logEvent, TextWriter output)
                 // rendering and support some additional formats: 'u' for uppercase
                 // and 'w' for lowercase.
                 var sv = propertyValue as ScalarValue;
-                if (sv != null && sv.Value is string)
```

Comment:
I'm not sure I completely understand why this change works - previously we only used this rendering path for strings, but in the new version we'll end up rendering everything - integers, datetimes and so-on, down this path. Am I misunderstanding here? Thanks!